### PR TITLE
Update oracle profile

### PR DESCRIPTION
**Pull request by aosingh**
## Description & motivation

- Moved dbt-oracle from community-built to vendor-built in available-adapters.md
- Updated oracle-profile.md

## To-do before merge
-  [ ] release dbt-oracle==1.0.0 on PyPI
-  [ ] publish release version on Github repo

### CODE REVIEW

File: website/docs/reference/warehouse-profiles/oracle-profile.md [hunk 1]
```diff
@@ -2,50 +2,188 @@
...
 
-**Maintained by:** Community    
-**Author:** Vitor Avancini     
-**Source:** [Github](https://github.com/techindicium/dbt-oracle)    
-**Core version:** v0.16.0 and newer      
+**Maintained by:** Oracle    
+**Source:** [Github](https://github.com/oracle/dbt-oracle)    
+**Core version:** v1.0.5     
 **dbt Cloud:** Not Supported    
 **dbt Slack channel** [Link to channel](https://getdbt.slack.com/archives/C01PWH4TXLY)
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/oracle-profile.md]:
```suggestion
**dbt Slack channel** [#db-oracle](https://getdbt.slack.com/archives/C01PWH4TXLY)       
```

File: website/docs/reference/warehouse-profiles/oracle-profile.md [hunk 2]
```diff
@@ -2,50 +2,188 @@
...
-![dbt-oracle stars](https://img.shields.io/github/stars/techindicium/dbt-oracle?style=for-the-badge)
+![dbt-oracle stars](https://img.shields.io/github/stars/oracle/dbt-oracle?style=for-the-badge)
+
+## Installation
 
-Easiest install is to use pip:
+dbt-oracle can be installed via the Python Package Index (PyPI) using pip
 
-    pip install dbt-oracle
+    pip install -U dbt-oracle
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/oracle-profile.md]: why add the `-U` option? That shouldn't be necessary in most cases.
```suggestion
    pip install dbt-oracle
```

File: website/docs/reference/warehouse-profiles/oracle-profile.md [hunk 3]
```diff
@@ -2,50 +2,188 @@
...
 
 You will need Oracle client driver installed. Check this [link](https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html) for the installation guide for your operating system
 
-### Connecting to Oracle with **dbt-oracle**
+## Connecting to Oracle Database
+
+Define the following mandatory parameters as environment variables and refer them in the connection profile using [env_var](https://docs.getdbt.com/reference/dbt-jinja-functions/env_var) jinja function. Optionally, you can also define these directly in the `profiles.yml` file.
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/oracle-profile.md]:
```suggestion
Define the following mandatory parameters as environment variables and refer them in the connection profile using [env_var](https://docs.getdbt.com/reference/dbt-jinja-functions/env_var) jinja function. Optionally, you can also define these directly in the `profiles.yml` file, but this is not recommended
```

File: website/docs/reference/warehouse-profiles/oracle-profile.md [hunk 4]
```diff
@@ -2,50 +2,188 @@
...
+export DBT_ORACLE_TNS_NAME=net_service_name
+```
+
+<File name='~/.dbt/profiles.yml'>
 
 ```yaml
-dbt_oracle:
-   target: dev
+dbt_test:
+   target: "{{ env_var('DBT_TARGET', 'dev') }}"
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/oracle-profile.md]: is `DBT_TARGET` still necessary to show in this example?
```suggestion
   target: dev
```

**Review by dataders — COMMENTED** [review]
This is a MUCH improved version of the previous doc! Thanks for all the detail showing how many different ways users can connect. I made a few comments, and if you want extra credit, you can try the Tabbing functionality that you can see on the [Synapse Profile page](https://docs.getdbt.com/reference/warehouse-profiles/azuresynapse-profile#active-directory-authentication) (done by me a long time ago in #423 :) )

**Review by dataders — APPROVED** [review]

**Review by dataders — APPROVED** [review]
I'm impressed you got the tabbing working -- it looks great and makes a big difference IMHO cheers @aosingh!

**Comment by aosingh** [comment]
Thank you @dataders for merging this! I agree, tabbing makes it easier to read and the content looks organized.

**Pull request merged**